release-notes/major10/10.0.0/migration.md

```diff
@@ -2,20 +2,29 @@
 
 This page describes how to update Valtimo from the previous version to the current.
 
-* **Breaking change 1/Deprecation 1**
+* **Node.js and NPM version upgrade**
```

Comment:
Please add  the scope as well. Ivo added this already to his PR by adding "Scope: Backend" just below the bullet point.